### DISCUSSION TITLE
Ask for help! An error was reported after using make modules during the firmware compilation process of esp32, but it failed according to the solution idea. The picture is as follows. You can write under this post or contact me at liheng2137@qq.com

### DISCUSSION THREAD

**Discussion by lmliheng** [micropython/micropython#15184] [ESP32]
<img width="488" alt="734bb1edee83f9bfc4b4a44cd281363" src="https://github.com/micropython/micropython/assets/159103134/3d536407-043d-4863-85eb-faec156a7b60">

**Comment by scruss** [comment]
Looks like a successful build to me

**Reply by lmliheng** [reply]
Okay, but I was wrong

**Comment by Josverl** [comment]
also please read the [userguide ](https://github.com/orgs/micropython/discussions/9111)to learn how to 
- create a clear short tile 
- paste and format code / logs , rather than screenshots

**Reply by lmliheng** [reply]
good
